product/product.go

```diff
@@ -110,12 +112,12 @@ func (p *Product) Filter() error {
 // CommanderHealthCheck employs the CLI to check if the client and then the agent are available.
 func CommanderHealthCheck(client, agent string) error {
 	checkClient := runner.NewCommander(client, "string", nil).Run()
-	if checkClient.Error != nil {
-		return fmt.Errorf("client not available, healthcheck=%v, result=%v, error=%v", client, checkClient.Result, checkClient.Error)
+	if checkClient[0].Error != nil {
```

Comment:
I'm thinking of maybe changing Op.Error (etc.) to Op.Errors() which makes the difference between multi-op runners (do/dosync groupings) and single-op runners (all others) transparent. That would save a lot of this ugly/brittle `op[0]` stuff.